docs/services/email/smtp/index.md

```
@@ -1,8 +1,244 @@
+
+    ```uri
+    smtp://user:pass@mail.example.com:587/?fromaddress=alerts@example.com&toaddresses=ops@example.com&timeout=30s
+    ```
+
+## Message Templates
+
+SMTP templates are a __Go library API only__. They are not URL or CLI parameters.
+
+IDs are `"plain"` and `"HTML"`. The template data map has a `message` key.
+
+```go
```

markdownlint-cli2 (0.23.2)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)